src/test/java/org/fundacionjala/pivotal/steps/ui/project/ProjectAssertionSteps.java

```diff
@@ -0,0 +1,123 @@
+package org.fundacionjala.pivotal.steps.ui.project;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import org.fundacionjala.pivotal.pages.account.Account;
+import org.fundacionjala.pivotal.pages.dashboard.DashBoard;
+import org.fundacionjala.pivotal.pages.project.CreateProject;
+import org.fundacionjala.pivotal.pages.project.DeleteProject;
+import org.fundacionjala.pivotal.pages.project.EditProject;
+import org.fundacionjala.pivotal.pages.project.PageLogin;
+import org.fundacionjala.pivotal.steps.FeatureNames;
+import org.fundacionjala.pivotal.util.Helper;
+import org.testng.Assert;
+import org.testng.asserts.Assertion;
+import org.testng.asserts.SoftAssert;
+
+/**
+ * Class that contains the assertion steps of Project feature.
+ */
+@SuppressWarnings("ALL")
+public class ProjectAssertionSteps {
+
+    private Account account;
+    private final Assertion assertion;
+    private EditProject editProject;
+    private DashBoard dashBoard;
+    private FeatureNames feature;
+    private PageLogin pageLogin;
+    private CreateProject createProject;
+    private FeatureNames proyName;
+    private DeleteProject deleteProject;
+
+    /**
+     * @param editProject   EditProject page object.
+     * @param dashBoard     Dashboard page object.
+     * @param deleteProject DeleteProject page object.
+     * @param account       EditProject page
+     * @param feature       Account page object.
+     * @param pageLogin     Login page object.
+     * @param createProject CreateProject page object.
+     * @param proyName      name of the project.
+     * @param helper        Helper object.
+     */
+
+    // CSOFF: ParameterNumber
+    public ProjectAssertionSteps(final EditProject editProject, final DashBoard dashBoard,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)